### ISSUE TITLE
scaling support

### ISSUE THREAD

**Issue by plumber-craic** [comictagger/comictagger#668]
When I'm at 200% scaling the UI is hard to use. At 100% though on this 4k laptop it's too small for me. Is there a workaround?
![image](https://github.com/user-attachments/assets/4641580c-2f7f-45ed-8ade-449ea8f287ba)
![image](https://github.com/user-attachments/assets/4b13e611-2436-4e76-9724-7fb2e7bb03a8)

**Comment by lordwelch** [comment]
Not really, I've never dealt with any UI scaling issues before. Both MacOS and Windows have scaling working with Qt, MacOS did need an option enabled in the .app .plist `NSHighResolutionCapable=True` but I've no idea what linux would need

**Comment by mizaki** [comment]
If it's linux, Xorg or Wayland, GNOME, KDE, etc.? I'm using KDE6.1 on Wayland and the scaling works correctly. 

Or do you mean an independent scaling to the system scaling?

**Comment by plumber-craic** [comment]
This is on linux mint 21 cinnamon, using Xorg. Still loving comictagger even though its hard to use when system is scaled above 100%.

**Comment by mizaki** [comment]
Are you using GNOME? My only other thought is if you've set GNOME to use scaling maybe KDE/Qt has it's scaling set to 100%. There doesn't appear to be scaling in QT Settings program so, not sure.
